boilerplate/src/app/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { observer } from "mobx-react-lite"
+import React from "react"
+import { Image, ImageStyle, TextStyle, View, ViewStyle } from "react-native"
+import { Text } from "src/components"
+import { isRTL } from "../i18n"
+import { colors, spacing } from "../theme"
+import { useSafeAreaInsetsStyle } from "../utils/useSafeAreaInsetsStyle"
+
+const welcomeLogo = require("../../assets/images/logo.png")
+const welcomeFace = require("../../assets/images/welcome-face.png")
+
+export default observer(function WelcomeScreen() {
+  const $bottomContainerInsets = useSafeAreaInsetsStyle(["bottom"])
+
+  return (
+    <View style={$container}>
+      <View style={$topContainer}>
```

Comment:
So I guess we'll need to remember that this screen & _layout are now duplicated, if we ever need to make updates. Not a bad tradeoff, just noteworthy for future maintenance.

Reply:
The `_layout.tsx` isn't but yeah `WelcomeScreen.tsx` -> `index.tsx` is, I don't love it. But it wasn't as simple as a rename. 

I thought coding more regex replace stuff was uglier to maintain over just moving it. We don't change the welcome screen that often but yeah we could cook up something else